.github/workflows/publish_pypi.yml

```diff
@@ -1,5 +1,5 @@
-# The cibuildwheel action triggers on creation of a release, this
-# triggers on publication.
+# The cibuildwheel action triggers on all pushes and PRs;
+# this action triggers on release publication.
```

Comment:
If we're changing this across the explosion ecosystem, maybe we should also change https://github.com/explosion/gha-cibuildwheel/blob/main/publish_pypi.yml.